README.md

```diff
@@ -13,7 +13,7 @@ For building & pushing the image locally:
 
 For local development:
 
-1. miniconda3-4.7.12 (can be installed via pyenv)
+1. miniconda3-24.7.1 (can be installed via pyenv)
```

Comment:
I'm puzzling over whether we need to be so strict on this. Seems like a bit of a burden. Maybe we should just say miniconda 24?